public/scss/pages/Article.module.scss

```
@@ -52,6 +52,16 @@
       color: $color_black_text;
       margin: 0;
     }
+
+    img{
+      max-width: 100%;
+      height: auto;
+    }
```

boleh dikasih enter mba?
```suggestion
    img
    {
      max-width: 100%;
      height: auto;
    }
```